db/migrate/20151017171417_rpush_2_0_0_updates.rb

```diff
@@ -0,0 +1,67 @@
+class Rpush200Updates < ActiveRecord::Migration
+  module Rpush
+    class App < ActiveRecord::Base
+      self.table_name = 'rpush_apps'
+    end
+
+    class Notification < ActiveRecord::Base
+      self.table_name = 'rpush_notifications'
+    end
+  end
+
+  def self.update_type(model, from, to)
+    model.where(type: from).update_all(type: to)
+  end
+
+  def self.up
+    add_column :rpush_notifications, :processing, :boolean, null: false, default: false
+    add_column :rpush_notifications, :priority, :integer, null: true
+
+    if index_name_exists?(:rpush_notifications, :index_rpush_notifications_multi, true)
+      remove_index :rpush_notifications, name: :index_rpush_notifications_multi
+    end
+
+    add_index :rpush_notifications, [:delivered, :failed], name: 'index_rpush_notifications_multi', where: 'NOT delivered AND NOT failed'
+
+    rename_column :rpush_feedback, :app, :app_id
+
+    if postgresql?
+      execute('ALTER TABLE rpush_feedback ALTER COLUMN app_id TYPE integer USING (trim(app_id)::integer)')
```

Comment:
Line is too long. [106/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.